lib/colorls/flags.rb

```diff
@@ -34,6 +36,52 @@ def process
 
     private
 
+    def add_sort_options(options)
+      options.separator ''
+      options.separator 'sorting options:'
+      options.separator ''
+      options.on('-sMODE', %w[d f], 'sort (f)iles or (d)irectories first') do |m|
+        @opts[:sort] = m == 'd' ? :dirs : :files
+      end
+      options.on('--sort-dirs', '--group-directories-first', 'sort directories first') { @opts[:sort] = :dirs }
+      options.on('--sort-files', 'sort files first')                                   { @opts[:sort] = :files }
+    end
```

Comment:
Please add `-sd` as short-hand of `--sort-dirs`, and similarly `-sf` for `--sort-files`